<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="headerTitle">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255); background-color: rgb(0, 24, 35);</string>
     </property>
     <property name="text">
      <string>PROFILE</string>
     </property>
     <property name="textFormat">
      <enum>Qt::TextFormat::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="IsiFrame">
     <property name="styleSheet">
      <string notr="true">.QFrame {
background-color: rgb(0, 33, 48);
}
QLineEdit, QTextEdit{
background-color: rgb(46, 47, 48);
}
QFrame{
font-weight: bold; color: #ffffff;
}

QCheckBox::indicator:unchecked{background-color : rgb(46, 47, 48);}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>30</number>
      </property>
      <property name="horizontalSpacing">
       <number>20</number>
      </property>
      <property name="verticalSpacing">
       <number>15</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="namaLabel">
        <property name="text">
         <string>Nama Lengkap </string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="emailLabel">
        <property name="text">
         <string>Username</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="passwordLabel">
        <property name="text">
         <string>Password          </string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="teleponLabel">
        <property name="text">
         <string>Telepon             </string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="jeniskelaminLabel">
        <property name="text">
         <string>Jenis Kelamin   </string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="lakiCBox">
        <property name="text">
         <string>Laki-laki</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">genderGrupBtn</string>
        </attribute>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QCheckBox" name="perempuanCBox">
        <property name="text">
         <string>Perempuan</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">genderGrupBtn</string>
        </attribute>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="alamatLabel">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="tabletTracking">
         <bool>false</bool>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LayoutDirection::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Alamat               </string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="2">
       <widget class="QTextEdit" name="alamatInput">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>80</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="namaInput">
        <property name="frame">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="userInput">
        <property name="frame">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLineEdit" name="passwordInput">
        <property name="text">
         <string/>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QLineEdit" name="teleponInput">
        <property name="frame">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="saveFrame">
     <property name="styleSheet">
      <string notr="true">.QFrame {
background-color: rgb(0, 33, 48);
}
                QPushButton {
					background-color: rgb(0, 80, 115);
                    font-size: 20px;
                    font-weight: bold;
                    color: #ffffff;
                    padding-left: 20px;
                    border: none;
                    height: 50px;
                }
                QPushButton:hover {
                    background-color: rgb(0, 50, 100);
                }</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="text">
         <string>SIMPAN PERUBAHAN</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="genderGrupBtn"/>
 </buttongroups>
</ui>
